src/covid_model_deaths/deaths_io/loctions.py

```diff
@@ -0,0 +1,37 @@
+from typing import NamedTuple
+
+from db_queries import get_location_metadata
+import pandas as pd
+
+
+class _LocationGroups(NamedTuple):
+    us: str = 'United States'
+    non_us: str = 'Not United States'
+
+
+LOCATION_GROUPS = _LocationGroups()
+
+
+def get_locations(location_group: str, location_set_version_id: int) -> pd.DataFrame:
+    loc_df = get_location_metadata(location_set_id=111,
+                                   location_set_version_id=location_set_version_id)
+    most_detailed = loc_df['most_detailed'] == 1
+    good_ids = ~loc_df['location_id'].isin([53474, 53451, 53452])
+    keep_columns = ['location_id', 'location_ascii_name', 'parent_id', 'level', 'most_detailed']
+
+    loc_df = loc_df.loc[most_detailed * good_ids, keep_columns]
```

Comment:
didn't know about this * syntax. Nice.

Reply:
Hah, I think this is an error that happens to still do what I want.  Bitwise & and vectorized * work the same on boolean masks.  Not intentional though.